src/main/resources/com/mathworks/ci/RunMatlabTestsStep/help-sourceFolder.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    <p>Specify the location of folders containing source code, relative to the project root folder. The specified folders and their subfolders are added to the top of the MATLAB search path. To generate a coverage report, MATLAB uses only the source code in the specified folders and their subfolders.</p>
+    <p>Paste the source folder path in the box in a list format.</p>
+    <p>Example: ["src/folderA", "src/folderB"]</p>
```

Comment:
This isn't true anymore, right?

Reply:
Which part? This is true for pipeline. 
This help file is for Pipeline Syntax Snippet Generator section.

Reply:
Oh I see. Thanks for the clarification.

Reply:
RE: Paste the source folder path in the box in a list format.

It's not clear what is meant by "in a list format". Can't we remove it?

Reply:
Hi @mw-hrastega , Here(Pipeline Syntax Snippet Generator) the help text is slightly different than the help text in FreeStyle Projects. Here there's a single textbox for entering a list of values for source folder.
I included an example for the same in the help text. 
Could you suggest a better way of putting it?